src/main/kotlin/step3/racing/printer/CarPrinterImpl.kt

```diff
@@ -0,0 +1,9 @@
+package step3.racing.printer
+
+import step3.racing.car.Car
+import step3.racing.mock.View
+
+class CarPrinterImpl(override val view: View) : CarPrinter {
+    override fun printString(car: Car) = (0 until car.distance)
+        .joinToString(separator = "") { "_" }
```

Comment:
`String`에 대해 호출할 수 있는 `repeat()`가 있습니다.

```suggestion
    override fun printString(car: Car) = "_".repeat(car.distance)
```